airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ResourceNotAvailable/ResourceNotAvailable.module.scss

```diff
@@ -0,0 +1,10 @@
+@use "scss/variables";
+
+.requestContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: variables.$spacing-lg;
+  justify-content: center;
+  height: 80vh;
```

Comment:
I think a bit hacky solution, and only "pseudo centering" it, and likely we should rewrite the documentation panel to actually use flex box, but I think that can be done in a later PR.